agent/src/main/java/com/teamscale/jacoco/agent/util/AgentUtils.java

```diff
@@ -23,7 +23,7 @@ public static Path getMainTempDirectory() {
 		if (mainTempDirectory == null) {
 			try {
 				mainTempDirectory = Files.createTempDirectory("teamscale-java-profiler-" +
-						FileSystemUtils.toSafeFilename(ProcessInformationRetriever.getPID()) + "-");
+						FileSystemUtils.toSafeFilename(ProcessInformationRetriever.getPID()));
```

Comment:
Please leave the `-` at the end. It separates the PID from the random string that is appended by Java to make the directory unique. Being able to quickly see the PID from the directory name may be useful during debugging to identify the temp directory (containing e.g. the log files) of a problematic process!

With the dash you get directories like `/tmp/teamscale-java-profiler-1234-15687546546573`
Without they look like `/tmp/teamscale-java-profiler-123415687546546573` where it's not clear what the PID was for that directory.

Reply:
Maybe add a comment with this rationale so others don't do the same thing again in the future? :P

Reply:
💡